lib/ex_force.ex

```diff
@@ -169,6 +169,20 @@ defmodule ExForce do
   end
 
   @doc """
+  Creates a SObject.
+
+  See [SObject Rows](https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/resources_sobject_retrieve.htm)
+  """
+  @spec create_sobject(sobject_name, map, config_or_func) :: :ok | {:error, any}
+  def create_sobject(name, attrs, config \\ default_config()) do
+    case request_post("/sobjects/#{name}/", Poison.encode!(attrs), config) do
+      {201, raw} ->
+        {:ok, extract_create_id(raw)}
```

Comment:
Could you run `mix format`? I believe the formatter will make single line vs multil line for pattern match in `case` consistent.

See https://elixirforum.com/t/running-the-new-elixir-formatter/9344/36 to install master elixir and run it, without interrupting your elixir version.

Reply:
ah, I have 1.5.2. I ll install a master release of elixir then  (I am using asdf for Elixir verison management so I ll figure it out). I heard for the formatter but did not try it yet :)